cardano-api/src/Cardano/Api/Tx.hs

```diff
@@ -521,9 +521,10 @@ makeSignedTransaction witnesses (ShelleyTxBody era txbody txmetadata) =
           (maybeToStrictMaybe txmetadata)
 
 
-makeByronKeyWitness :: NetworkId
+makeByronKeyWitness :: forall key. IsByronKey key
```

Comment:
Usually we give the forall it's own line, with the class constraints starting on the next line.